docs/fields.asciidoc

```diff
@@ -93,6 +93,27 @@ A flat mapping of user-defined tags with string values.
 --
 
 
+*`context.http.method`*::
++
+--
+type: keyword
+
+The method of the http request.
+
+
+--
+
+*`context.http.status_code`*::
++
+--
+type: long
+
+The status code of the http request.
```

Comment:
nitpick: this is actually the HTTP response.

i am also seeing that we write indifferently HTTP and http, maybe you can use this change to grep and replace to all caps everywhere.